oslabs-beta/React-Labyrinth
# AL/directives

## Overview

**Issue Type**


**Description**
Added conditionals for checking directives such as 'use client' to determine the component type.
Added 'npx tsc ; -path to test build-' in package.json

**Ticket Item**
RL-36: Create conditional for 'use client'/'use server' in parser

**Steps to Validate Feature**

1. Run `npm run webpack`
2. Run extension test
3. Pass in the file with 'use client' directive or without
4. See accurate changes in component typing 

**Previous behavior**
Before we had a conditional on rendering component type based on the use of hooks.

**Expected behavior**
Now we have conditional on rendering component type based on use of directives and hooks.
